api/turing/service/pagination.go

```diff
@@ -29,3 +33,22 @@ func PaginationScope(options PaginationOptions) func(db *gorm.DB) *gorm.DB {
 		return db
 	}
 }
+
+// CreatePaginatedResults is a helper function that helps to create paginated results
+func CreatePaginatedResults(options PaginationOptions, count int, results interface{}) *PaginatedResults {
```

Comment:
Good idea 👍. But this probably can be package-private, don't expect it to be used outside of `service` package

Reply:
done